docs/test_scripts/domain_parallelism/new_layers/reshape_subtract.py

```diff
@@ -0,0 +1,41 @@
+import torch
+import torch.distributed as dist
+import time
+
+from physicsnemo.distributed import DistributedManager, scatter_tensor, ShardTensor
+from torch.distributed.tensor.placement_types import Shard, Replicate
+
+from physicsnemo.distributed.shard_utils.ring import perform_ring_iteration, RingPassingConfig
+
+# This time, let's make two moderately large tensors since we'll have to, at least briefly,
+# construct a tensor of their point-by-point difference.
+N1 = 234_567
+N2 = 12_345
+num_neighbors = 17
+
+DistributedManager.initialize()
+dm = DistributedManager()
+
+# We'll make these 3D tensors to represent 3D points
+a = torch.randn(N1, 3, device=dm.device)
+b = torch.randn(N2, 3, device=dm.device)
+
+mesh = dm.initialize_mesh([-1,], ["domain"])
+placements = (Shard(0),)
+a_sharded = scatter_tensor(a, 0, mesh, placements)
+b_sharded = scatter_tensor(b, 0, mesh, placements)
```

Comment:
In general, when it comes to tutorial scripts that are introducing new functions, I would lean towards keyword args when possible - it makes it far easier for a new user to understand what's happening. Positional arguments save time for the writer but can cost time for the reader (if it's not a function they're already well-acquainted with, like `torch.randn`).

To be specific: here, it would be useful to explain to the reader what the placements argument does, and how it is formatted (e.g., why is it a length-1 tuple, what does the `0` in `Shard(0)` do, etc.) - this is not necessarily obvious here

Reply:
Resolved, thanks!